lib/views.js

```diff
@@ -418,4 +418,51 @@ exports.doc_summaries_by_id = {
       });
     }
   }
-};
\ No newline at end of file
+};
+
+exports.doc_by_place = {
+  map: function(doc) {
+
+    var emitPlace = function(place) {
+      if (!place) {
+        emit([ '_unassigned' ]);
+        return;
+      }
+      while (place) {
+        emit([ place._id ]);
+        place = place.parent;
+      }
+      return;
+    };
+
+    if (doc._id === '_design/medic') {
```

Comment:
Yeah true. Initially it was added to the filter as a safety because the ddoc kept sneaking through and caused real problems. But it's better to put this in a unit test than bloat the map function.